modules/backup-azure/client.go

```diff
@@ -215,6 +219,42 @@ func (a *azureClient) WriteToFile(ctx context.Context, backupID, key, destPath s
 	return nil
 }
 
+func (a *azureClient) getBlockSize(ctx context.Context) int64 {
+	blockSize := int64(40 * 1024 * 1024)
```

Comment:
Can we make it as `const`?

Reply:
That's a good idea then this const could be used here and in line 233. @donomii could you add it?

Reply:
ok. done